schema/MTMount.yaml

```diff
@@ -20,8 +20,13 @@ properties:
     type: number
     exclusiveMinimum: 0
     default: 10
+  camera_cable_wrap_advance_time:
+      description: How far in advance to make the tai time field of camera cable wrap tracking commands (sec)
```

Comment:
I recommend to change the description to: "Estimated delay in seconds between MTMount receiving rotator trajectory and Camera Cable Wrap (CCW) hardware applying the new trajectory to cable wrap. Trajectory data send to CCW are adjusted to have position calculated at the current time + this value. Expectations are for a small value (few ms). Default to 0.2 (seconds).

Reply:
I agree that the description needs work, but do not want to recommend a specific value until we do more testing. (I also hope that the TMA can be adjusted to extrapolate for a short time if tracking commands are late, but such a change may cost more than we can afford and we've not asked for it yet).

I have pushed an updated description that explains things in much more detail.